JS DOM Basic Assignments/Counter/script.js

```diff
@@ -0,0 +1,42 @@
+let counter = document.querySelector('.counter');
+const addCount = document.querySelector('#addCntBtn');
+const decCount = document.querySelector('#lowerCntBtn');
+let count = 0;
+addCount.addEventListener('click', incrementCount);
+decCount.addEventListener('click', decrementCount);
+
+function incrementCount() {
```

Comment:
convert this to es6 function

Reply:
So will need to declare const function before call
Correct ?